### ISSUE TITLE
Find out which entities are stable within Bonita

### ISSUE THREAD

**Issue by blcham** [kbss-cvut/bpmn2stamp#23]
It seems that IDs consist of 2 parts (see diff file).

A/C:
- we know which entites are changing
- we know how Id-s are generated

**Comment by grigobog** [comment]
Screenshot of difference:

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/62146890/168598698-e7c185ba-9ffc-4fa9-be28-7c0f9f54a03b.png">

**Comment by grigobog** [comment]
Before:
[timer-test.bpmn.txt](https://github.com/kbss-cvut/bpmn2stamp/files/8700210/timer-test.bpmn.txt)
After:
[timer-test-updated.bpmn.txt](https://github.com/kbss-cvut/bpmn2stamp/files/8700209/timer-test-updated.bpmn.txt)

**Comment by grigobog** [comment]
Most item identifiers are generated and stored in a .proc file in Bonita. These identifiers are used when exporting to .bpmn format is being done, which means that the identifiers remain unchanged, making the export stable within Bonita.
However, there are some items that are missing in the .proc file and appear in the .bpmn file after export (e.g. model:performer). A new random identifier is generated for these items.
For current state of the bpmn2stamp converter it is not a problem, since such elements are not used during conversion.

- we know which entites are changing
-- basically entities, which are not in proc file. Another way to check is to export to .bpmn file two times in a row and compare the results.
- we know how Id-s are generated
-- it's possible to find out how exactly ids are generated during .bpmn export, but it will take some time to analyze Bonita code. In my opinion it is not needed now, since we can always verify what elements are affected and not to use such elements.
